hugo/content/pgo-client/common-tasks.md

```diff
@@ -291,6 +291,29 @@ created Pgcluster hacluster
 workflow id ae714d12-f5d0-4fa9-910f-21944b41dec8
 ```
 
+#### Create a PostgreSQL Cluster with Different PVC Sizes
+
+You can also create a PostgreSQL cluster with an arbitrary PVC size using the
+[`pgo create cluster`](/pgo-client/reference/pgo_create_cluster/) command. For
+example, if you want to create a PostgreSQL cluster with with a 128GB PVC, you
```

Comment:
Not to be pedantic, but wouldn't it be 128GiB (Gibibytes) given the notation below?

Reply:
Sure